core/src/main/scala/org/apache/spark/eventhubscommon/rdd/EventHubsRDD.scala

```diff
@@ -82,6 +84,7 @@ private[spark] class EventHubsRDD(
       cnt += 1
       receivedBuffer ++= receivedEvents
     }
+    logInfo(s"Finished ingesting events from partition ${eventHubNameAndPartition.partitionId}")
```

Comment:
Why it is necessary, we have https://github.com/sabeegrewal/spark-eventhubs/blob/9e497a4f7d5382fcdbc33a6ba1e409581c088996/core/src/main/scala/org/apache/spark/eventhubscommon/rdd/EventHubsRDD.scala#L120